ConnectionManager: startup fails when PostgreSQL username contains a dot

## Description
  
  When the PostgreSQL username contains a dot (e.g. `username.scorpio`), the application fails to start with:

  Caused by: io.vertx.pgclient.PgException: ERROR: syntax error at or near "." (42601)

  ## Root Cause
  
  `ConnectionManager.setup()` builds the following SQL by string concatenation:

  ```java
  executeQuery(null, "ALTER USER " + dbUser + " SET jit = off;", null, false)
```
  When dbUser is username.scorpio, the resulting SQL is:
```sql
  ALTER USER username.scorpio SET jit = off;
  ```
  PostgreSQL interprets the . as a schema separator (schema.object), which is invalid syntax in this context, causing the startup failure.

  Steps to Reproduce
  
  1. Configure scorpio.postgres.username (or DBUSER env var) to a value containing a dot, e.g. username.scorpio
  2. Start the application
  3. Startup fails with PgException: ERROR: syntax error at or near "."

  Fix

  Replace the dynamic username with CURRENT_USER, which is a SQL keyword automatically resolved by PostgreSQL to the connected user:
```java
  executeQuery(null, "ALTER USER CURRENT_USER SET jit = " + jitValue, null, false)
  ```

  Environment
  
  - Scorpio version: 6.0.1
  - PostgreSQL: 15
  - Deployment: Kubernetes

--- a/Commons/src/main/java/eu/neclab/ngsildbroker/commons/storage/ConnectionManager.java
+++ b/Commons/src/main/java/eu/neclab/ngsildbroker/commons/storage/ConnectionManager.java
@@ -85,9 +85,6 @@ public class ConnectionManager {
 	@ConfigProperty(name = "pool.initialSize")
 	int initialSize;
 
-	@ConfigProperty(name = "scorpio.postgres.username")
-	String dbUser;
-
 	@ConfigProperty(name = "scorpio.postgres.disablejit", defaultValue = "true")
 	boolean disableJIT;
 
@@ -95,11 +92,8 @@ public class ConnectionManager {
 
 	@PostConstruct
 	void setup() throws URISyntaxException {
-		if (disableJIT) {
-			executeQuery(null, "ALTER USER " + dbUser + " SET jit = off;", null, false).await().indefinitely();
-		} else {
-			executeQuery(null, "ALTER USER " + dbUser + " SET jit = on;", null, false).await().indefinitely();
-		}
+		String jitValue = disableJIT ? "off" : "on";
+		executeQuery(null, "ALTER USER CURRENT_USER SET jit = " + jitValue, null, false).await().indefinitely();
 
 		URI uri = new URI(reactiveDefaultUrl);
 		reactiveBaseUrl = uri.getScheme() + "://" + uri.getHost() + ":" + uri.getPort() + "/";

